public/index.js

```diff
@@ -0,0 +1,7 @@
+var search_box = document.querySelector('#nav_search_box');
+search_box.addEventListener('change', search);
+function search(event){
+    var url='/search/'+search_box.value.replace(/\s+/g, '').toLowerCase();
+    console.log('URL requested:', url);
+    window.location.replace(url);
```

Comment:
I think we should use a different function than replace. According to Mozilla replace breaks hitting the back button  https://developer.mozilla.org/en-US/docs/Web/API/Location/replace . I think we should change this to assign https://www.w3schools.com/jsref/met_loc_assign.asp.

Reply:
Done